### DISCUSSION TITLE
Is there an equivalent TERMUX_PKG_HOSTBUILD_DEPENDS?

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#19528] [Q&A]
Hi. I have to host-build a package requiring some dependencies (like boost). Is there an equivalent way to download those dependencies? Can I just `apt install package` inside the build script?

**Comment by Biswa96** [comment]
Where are you trying to build the package - in Android device or in PC/container?

**Reply by ghost** [reply]
I'm the building it in github actions termux provides.

**Reply by ghost** [reply]
The package executes it's own compiled binary which is unsuitable for the host. Error gives, "qemu-aarch64-static: Could not open '/system/bin/linker64': No such file or directory".

I figured out I need to build it for the host, but it requires boost dependencies.

**Comment by Biswa96** [comment]
TERMUX_PKG_BUILD_DEPENDS variable can be used. There is no such variable named TERMUX_PKG_HOSTBUILD_DEPENDS.
